src/cluster_legacy.c

```diff
@@ -3128,39 +3128,7 @@ int clusterProcessPacket(clusterLink *link) {
             clusterUpdateSlotsConfigWith(sender_primary, senderConfigEpoch, hdr->myslots);
 
         /* Explicitly check for a replication loop before attempting the replication
-         * chain folding logic.
```

Comment:
the comment is outupdated so we remove it? (i did not bother to read it...)

Reply:
Yes this particular circular replicaof chain is fixed